### DISCUSSION TITLE
postgres ownership?

### DISCUSSION THREAD

**Discussion by wuast94** [nextcloud/all-in-one#2649] [Questions] [answered]
I moved my server and fucked up my ownerships and permissons, postgres container dont start, logs say "FATAL:  data directory "/var/lib/postgresql/data" has wrong ownership"

tried to chmod -R root:root and postgres:postgres on the data and backup data dir of database but no luck.

anyone know what the right ownership is? or what i am missing=

**Comment by Zoey2936** [comment, marked as answer]
please try chown 999:999
